app/src/main/res/drawable/circular_cell.xml

```diff
@@ -5,5 +5,5 @@
             <solid android:color="@color/colorIcons"/>
         </shape>
     </item>
-    <item android:drawable="@drawable/cell" />
+    <item android:drawable="@drawable/cell" android:top="10dp" android:bottom="10dp" android:left="10dp" android:right="10dp" />
```

Comment:
@brianmwadime this technique seems to work, but not sure it will work on all/most screens. Do you know?